dipy/io/stateful_tractogram.py

```diff
@@ -110,11 +131,45 @@ def __init__(self, streamlines, reference, space,
             raise ValueError('Space MUST be from Space enum, e.g Space.VOX')
         self._space = space
 
+        if origin_at_corner in Origin:
+            origin_at_corner = origin_at_corner.value
+
         if not isinstance(origin_at_corner, bool):
-            raise TypeError('origin_at_corner MUST be a boolean')
+            raise TypeError('origin_at_corner MUST be a boolean or from '
+                            'Origin enum, e.g Origin.NIFTI')
         self._origin_at_corner = origin_at_corner
         logger.debug(self)
 
+    @staticmethod
+    def from_sft(streamlines, sft,
+                 data_per_point=None,
+                 data_per_streamline=None):
+        """ Re-create a strict, state-aware, robust tractogram from another
```

Comment:
```suggestion
        """ Create an instance of `StatefulTractogram` from another instance of `StatefulTractogram`.
```

Unless I am missing something: what does "re-create" mean in this context?